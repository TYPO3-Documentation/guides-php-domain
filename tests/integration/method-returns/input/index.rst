=============================
PHP methods with return value
=============================


..  php:method:: getSomething()
    :returns: Something not specified...

    Lorem Ipsum
