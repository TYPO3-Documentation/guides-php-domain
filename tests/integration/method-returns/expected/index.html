<!-- content start -->
    <div class="section" id="php-methods-with-return-value">
            <h1>PHP methods with return value</h1>
            <dl class="php method">
    <dt class="sig sig-object php" id="getsomething">
<span class="sig-name descname"><span class="pre">getSomething</span></span>
<span class="sig-paren">(</span>
<span class="sig-paren">)</span>
 <a class="headerlink" href="#getsomething" title="Permalink to this definition">Â¶</a>
    </dt>
    <dd>

    <p>Lorem Ipsum</p>

                    <dl class="field-list simple">
                <dt class="field-even">Returns:</dt>
                <dd class="field-even">
                    Something not specified...
                </dd>
            </dl>
        </dd>
</dl>
    </div>
<!-- content end -->
